.travis.yml

```diff
@@ -1,12 +1,16 @@
 language: python
 sudo: false
 python: 3.5
+before_install:
+  - nvm install node
```

Comment:
No biggie right now, but we should probably pick a version of node to build against at some point.